src/EmailCardList/EmailCard/EmailCard.js

```diff
@@ -0,0 +1,15 @@
+const EmailCard = (props) => {
+    return (
+        <div className={(props.read === true ? 'btn btn-light col-md-12 border-bottom' : 'btn btn-secondary border-bottom col-md-12') + " d-flex flex-row justify-content-between p-2"}>
```

Comment:
i think the border should be both on top & bottom, and also an increasing number of classes are being placed inside the ternary despite the fact that the only class being changed by the logic is `btn-light`/`btn-dark`

Reply:
this is why it should be on top and bottom
<img width="535" alt="image" src="https://user-images.githubusercontent.com/25844355/187710334-8cc45b5b-98c1-4988-b071-cfbc2a792818.png">

Reply:
I have removed everything except light and dark from the ternary
Border bottom impacts both the top and bottom of every box bar the very top one as the bottom of one box is the top of another